src/components/utils/mappings/mapTripToCarbonActivityTypeToDonutChartLabel.js

```diff
@@ -0,0 +1,12 @@
+const mappingObject = {
+  anyCar: 'car',
+  anyFlight: 'plane',
+  transitRail: 'train',
+  bus: 'bus',
+}
+
+export function mapTripToCarbonActivityTypeToDonutChartLabel(
```

Comment:
Der Name hat mich etwas verwirrt 😅   Eigentlich schaust du doch hier, welche Trip Types es überhaupt schon gibt und setzt nur dann die entsprechenden Label in der Legend, oder? Wie wäre der Name mapTripTypesToDonutChartLegend oder so?

Reply:
Übrigends spannend zu sehen wie man über ein Object mappen kann! Danke dafür :)